crates/pop-cli/src/main.rs

```diff
@@ -83,6 +164,24 @@ fn cache() -> Result<PathBuf> {
 	Ok(cache_path)
 }
 
+fn init_config() -> Result<()> {
+	let path = config_file_path()?;
+	match pop_telemetry::write_default_config(&path) {
+		Ok(written) => {
+			if written {
+				log::info(format!("Initialized config file at {}", &path.to_str().unwrap()))?;
```

Comment:
Why info? Seems more like a debug level.

Reply:
This is a cliclack log. Made more explicit

Reply:
Should it not then read `"Initialized telemetry config file at ..."` instead?